js/loginform.js

```diff
@@ -0,0 +1,84 @@
+const emailInput = document.querySelector("#email");
+const emailInputBox = document.querySelector(".input-email");
+const passwordInput = document.querySelector("#password");
+const passwordInputBox = document.querySelector(".input-password");
+const loginButton = document.querySelector(".login-button");
+const nicknameInput = document.querySelector("#nickname");
+const nicknameInputBox = document.querySelector(".input-nickname")
+const passwordConfirmInput = document.querySelector("#password-confirm");
+const passwordConfirmInputBox = document.querySelector(".input-password-confirm");
+const signupLoginButton = document.querySelector(".signup-login-button");
+const togglePasswordButton = document.querySelector("#toggle-password-visibility");
+const eyeIcon = document.querySelector("#eye-icon");
+
+// Email 유효성 검사
+function checkEmail(email) {
+    const emailPattern = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-za-z0-9\-]+/;
+    return emailPattern.test(email);
+};
+
+// Email 미입력 시 && Email 형식 오류 시
+emailInput.addEventListener("focusout", function(e){
+    if(!e.target.value){
+        emailInputBox.classList.add("error");
+        emailInputBox.classList.remove("errorform");
+    }else{
+        emailInputBox.classList.remove("error");
+        if (!checkEmail(e.target.value)){
+            emailInputBox.classList.add("errorform");
+        } else {
+            emailInputBox.classList.remove("errorform");
+        }
+    }
+});
+
+// Password 미입력 시 && 8글자 이하 입력 시
+passwordInput.addEventListener("focusout", function(e){
+    if(!e.target.value){
+        passwordInputBox.classList.add("error");
+        passwordInputBox.classList.remove("error8");
+    }else{
+        passwordInputBox.classList.remove("error");
+        if(e.target.value.length < 8) {
```

Comment:
비밀번호가 지금은 8글자까지인데 이후에 바뀔 수 있고, 단순한 숫자값이기에 상수를 정의해서 써주시면 더 좋을 거 같아요!

```suggestion
        const MINIMUM_PW_LENGTH = 8
       ...
        if(e.target.value.length < MINIMUM_PW_LENGTH) {
```